ddsc/core/util.py

```diff
@@ -181,3 +181,17 @@ def wait_for_processes(processes, size, progress_queue, watcher, item):
             raise ValueError(error_message)
     for process in processes:
         process.join()
+
+
+def verify_terminal_encoding():
+    """
+    Raises ValueError with error message when terminal encoding is ASCII.
+    """
+    if sys.stdout.encoding.endswith("-ASCII"):
```

Comment:
This misses several cases, and should probably check for UTF explicitly (rather than only failing if the string ends with ASCII)

Some examples:
If I set Text Encoding to __Western (Mac OS Roman)__, `LANG` is __en_US__ and `sys.stdout.encoding` becomes  `None`:
```
$ LANG=en_US python -c "import sys;print sys.stdout.encoding"
None
```

This would cause `verify_terminal_encoding()` to raise an exception.

If I set my encoding to __Western (ISO Latin 1)__, `LANG` is __en_US.ISO8859-1__, and `sys.stdout.encoding` is `ISO8859-1`:

Now, ISO8859-1 can still encode `u'\xa0'`, but it's probably lossy for other unicode characters...

Probably good to try under a Linux GUI environment too.